src/scene/timeframe.cpp

```diff
@@ -35,6 +35,12 @@
 #include <ghoul/misc/templatefactory.h>
 
 namespace {
+    // A TimeFrame object determines the time frame during which a scene graph node is
```

Comment:
```suggestion
    // A `TimeFrame` object determines the time frame during which a scene graph node is
```
again, is it an object? 🤔